packages/suite/src/views/firmware/index.tsx

```diff
@@ -34,15 +34,18 @@ const Wrapper = styled.div<{ isWithTopPadding: boolean }>`
 const StyledModal = styled(Modal)`
     width: 620px;
     min-height: 500px;
-    ${Modal.Body} {
+
+    > ${Modal.Body} {
         padding: 0;
         margin-top: 0;
         height: 100%;
+
         > * {
             height: 100%;
         }
     }
-    ${Modal.Content} {
+
+    > ${Modal.Content} {
```

Comment:
`Content` is not direct child of `Modal`, so this rule is currently unused.

Reply:
I didn't notice it affecting anything, removed 

cbd767025bd9279bc2d96c7ad5381c64bd6c3767

Reply:
Also no, it should not look like this 😭  Apparently it was long before this PR though...

cbd767025bd9279bc2d96c7ad5381c64bd6c3767